05/Weather.js

```diff
@@ -0,0 +1,20 @@
+import React from "react";
+import ReactDOM from "react-dom";
+
+class Weather extends React.Component {
+    constructor(props) {
+        super(props);
+    }
+
+    render() {
+        return (
+            <p>
+                W miescie {this.props.city} jest obecnie{" "}
+                {this.props.conditions.toLowerCase()}, a temperatura wynosi{" "}
+                {this.props.temp}
+            </p>
+        );
+    }
+}
```

Comment:
Lepiej w tym komponencie zrobić obsługę API bo łatwiej to będzie potem przenieść do innego projektu ;)
Jak już chcemy rozdzielić całkowicie to zapytania do API można trzymać w osobnym pliku.